Assets/Scripts/Player/PlayerCentral.cs

```diff
@@ -182,9 +182,9 @@ IEnumerator Dash()
         }
     }
 
-    IEnumerator Cooldown()
+    IEnumerator dashCooldown()
     {
-        yield return new WaitForSeconds(dashCooldown);
+        yield return new WaitForSeconds(dashCD/SpeedManager.playerMovementScaling);
```

Comment:
If we go on dash cd at 10% hp, then immediately after heal to full, this will still take 30s for the dash to come back online.

Reply:
updated